### PR TITLE
feat: add README.FR.md for French speakers

### PR DESCRIPTION
**Pull request by AlexShadow3**
## Added a README.FR.md for French Speakers

I have added a README.FR.md file to provide documentation for French-speaking users. This file is a translation of the existing README files, with a focus on ensuring clarity and readability in French.

### Purpose:
- To make the project more accessible to French speakers by offering a localized version of the documentation.
- To maintain consistency with the existing README files while adapting for linguistic and cultural nuances in French.

### Details:
- The content has been carefully translated to ensure it remains accurate and easy to understand.
- I also verified that all links, instructions, and code examples work correctly in this file.

Feel free to share any feedback or suggestions to further improve this translation!

### CODE REVIEW

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey @AlexShadow3 

Thank you so much for this. 

It seems this is based off of an older version of the README.  Could you base your translation on the README.md in the main branch, please?

**Comment by AlexShadow3** [comment]
Sure!

By the way, can I also add the French versions of CONTRIBUTING.md and CODE_OF_CONDUCT.md ?

**Comment by ogabrielluiz** [comment]
Hey @AlexShadow3 

Sure, go ahead. 

Let's keep this PR limited to the README, though. 

Thanks again!

File: README.zh_CN.md [hunk 1]
```diff
@@ -31,7 +31,7 @@
   <a href="./README.ES.md"><img alt="README in Spanish" src="https://img.shields.io/badge/Spanish-d9d9d9"></a>
   <a href="./README.zh_CN.md"><img alt="README in Simplified Chinese" src="https://img.shields.io/badge/简体中文-d9d9d9"></a>
   <a href="./README.ja.md"><img alt="README in Japanese" src="https://img.shields.io/badge/日本語-d9d9d9"></a>
-  <a href="./README.KR.md"><img alt="README in KOREAN" src="https://img.shields.io/badge/한국어-d9d9d9"></a>
+  <a href="./README.FR.md"><img alt="README in French" src="https://img.shields.io/badge/Français-d9d9d9"></a>
```
Comment (ogabrielluiz) [inline README.zh_CN.md:34]: Maybe you meant to replace the zh_CN instead of the KR?

Comment (AlexShadow3) [reply README.zh_CN.md:34]: Oh, I'm sorry,

I meant to add the line like in the other README files 🤦

**Comment by AlexShadow3** [comment]
Hey @ogabrielluiz

It might be good for now !

Do not hesitate to tell me if there are any errors left.

**Review by ogabrielluiz — APPROVED** [review]

**Comment by AlexShadow3** [comment]
Hey @ogabrielluiz 

I don't know how, but the original code of conduct was replaced by the French code.

image

**Comment by ogabrielluiz** [comment]
I don't think we can have many codes of conduct, then. Could you recreate the PR, please?

**Pull request merged**